src/transformers/models/opt/tokenization_opt.py

```diff
@@ -0,0 +1,417 @@
+# coding=utf-8
```

Comment:
Let's delete this file

Reply:
resolved in b931db8efbe7e79c7da2114017fe587265e2007e